pkg/controllers/templating_reconciler.go

```diff
@@ -165,7 +184,7 @@ func (r *TemplatingReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	return ctrl.Result{RequeueAfter: r.longWait}, errors.Wrap(r.kube.Status().Update(ctx, cr), errUpdateResourceStatus)
```

Comment:
Does the implementation of `onlyLog` log a stack trace? I would normally expect to see the error [logged as a structured key](https://github.com/crossplane/crossplane/blame/14a39bb783a18aad48711ae3a35a46b3d642e69a/docs/observability-developer-guide.md#L166). The stack trace is going to be helpful for anyone trying to debug some unexpected errors, so they can find the relevant code path more easily.

Reply:
> Does the implementation of onlyLog log a stack trace?

I am not sure, it just uses the logger that crossplane runtime provides. Just changed it to the structured key log.